.github/workflows/changesets.yml

```diff
@@ -24,6 +24,13 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
+      - name: Get Release token
+        id: release_token
+        uses: peter-murray/workflow-application-token-action@v2
```

Comment:
I think there's a v3 of this action available, any reason not to use it? 

https://github.com/marketplace/actions/workflow-application-token-action